src/main/java/mate/academy/mybookshop/dto/CreateCategoryRequestDto.java

```diff
@@ -0,0 +1,6 @@
+package mate.academy.mybookshop.dto;
+
+import jakarta.validation.constraints.NotBlank;
+
+public record CreateCategoryRequestDto(@NotBlank String name, String description) {
```

Comment:
Add max constraints to align with db constraints

Reply:
added